cernopendata/config.py

```diff
@@ -55,6 +55,7 @@
 #: Switch of automatic building.
 # ASSETS_AUTO_BUILD = True
 
+ACCOUNTS_REGISTER_BLUEPRINT = False
```

Comment:
this is needed eventually as the default is None and it is checked here https://github.com/inveniosoftware/invenio-accounts/blob/04d5171d9e508835f4c54998e4f327443929b75d/invenio_accounts/ext.py#L147

Reply:
Thanks @zzacharo. Let me create a new PR only with this change then.